.github/AUTODOC_GUIDE.md

```diff
@@ -0,0 +1,108 @@
+# dmdoc
+[DOCUMENTATION]: http://codedocs.tgstation13.org
+
+[BYOND]: https://secure.byond.com/
+
+[DMDOC]: https://github.com/SpaceManiac/SpacemanDMM/tree/master/src/dmdoc
+
+[DMDOC] is a documentation generator for DreamMaker, the scripting language
+of the [BYOND] game engine. It produces simple static HTML files based on
+documented files, macros, types, procs, and vars.
+
+We use **dmdoc** to generate [DOCUMENTATION] for our code, and that documentation
+is automatically generated and built on every new commit to the master branch
+
+This gives new developers a clickable reference [DOCUMENTATION] they can browse to better help
+gain understanding of the /tg/station codebase structure and api reference.
+
+## Documenting code on /tg/station
+We use block comments to document procs and classes, and we use `///` line comments
+when documenting individual variables.
+
+It is required that all new code be covered with DMdoc code, according to the [Requirements](#Required)
+
+We also require that when you touch older code, you must document the functions that you
+have touched in the process of updating that code
```

Comment:
What about the case of people refactoring some code but with some procs untouched, such that the documentation they do to abide by the requirements is scattered around, even though the system still uses unmodified undocumented procs? So you would see the main datum documented, a proc to handle some part after a timer, but a primary action proc that the sub proc calls is undocumented because "I didn't touch it!"

Reply:
that is okay, because it's better to have some documentation than all of it.

People are most likely to know what a proc does if they're touching it for a refactor that changes it's layout.